webpush_channels/tests/test_views_channel.py

```diff
@@ -115,17 +116,20 @@ def setUp(self):
         self.webpusher_error_patcher = mock.patch('webpush_channels.views.channels.WebPusher')
 
     def test_push_notifications_can_be_sent_to_channel_with_registration_and_subscription(self):
+        data = "You have a message"
         with self.webpusher_error_patcher as webpusher_mock:
             self.app.post(self.channel_url, headers=self.headers, status=202)
             webpusher_mock.assert_called_with(self.subscription)
+            webpusher_mock.return_value.send.assert_called_with(
+                data=json.dumps(data), ttl=15)
 
     def test_push_notification_can_take_a_payload(self):
         with self.webpusher_error_patcher as webpusher_mock:
             self.app.post_json(self.channel_url, MINIMALIST_PAYLOAD,
                                headers=self.headers, status=202)
             webpusher_mock.assert_called_with(self.subscription)
             webpusher_mock.return_value.send.assert_called_with(
-                data=MINIMALIST_PAYLOAD['data'], ttl=15)
+                data=json.dumps(MINIMALIST_PAYLOAD['data']), ttl=15)
```

Comment:
This can probably randomly fail because dict are not always using the same dict order.